projects/plugins/jetpack/extensions/blocks/videopress/v6/edit.js

```diff
@@ -225,7 +248,18 @@ export default function VideoPressEdit( { attributes, setAttributes } ) {
 					// eslint-disable-next-line no-console
 					console.error( 'Error: ', error );
 				} }
-			/>
+			>
+				{ uploadErrorData && (
+					<div
+						role="alert"
+						aria-live="assertive"
+						className="jetpack-videopress-upload-error-message"
+					>
+						{ uploadErrorData?.data?.message ??
```

Comment:
we should confirm whether the message from the response is translated. I guess so, but we never know.

Reply:
It is, in theory, translated in the server side. I suppose according to the cache site settings.

I  know for a fact that the strings in the server are in i18n functions